ondewo/nlu/intent.proto

```diff
@@ -940,32 +940,55 @@ message IntentSorting {
     SortingMode sorting_mode = 2;
 }
 
-message CreateTrainingPhraseRequest {
+message UpdateTrainingPhraseBatchRequest {
+    repeated Intent.TrainingPhrase training_phrases = 1;
+}
+
+message TrainingPhraseStatus {
+    oneof phrase_or_status {
+        Intent.TrainingPhrase training_phrase = 1;
+        string error_message = 2;
+    }
+}
+
+message TrainingPhraseStatusBatch{
+    repeated TrainingPhraseStatus training_phrase_statuses = 1;
+}
+
+message CreateTrainingPhraseBatchRequest {
     // Required. The name of the intent for which to create a training phrase for.
     // Needed since the name of the training phrase may not be set upon creation.
     // Format: `projects/<Project ID>/agent/intents/<Intent ID>`.
-    string intent_name = 1;
+    message CreatePhraseRequest{
```

Comment:
I get your intention, but your naming still forced me to think twice. Since you deleted my CreateTrainingPhraseRequest message, couldn't you just take over this naming here?

Reply:
I since this message is only used inside of `CreateTrainingPhraseBatchRequest` the context is clear, therefore i decided to make it shorter removing word `Training`

Reply:
It is only used there for definition of the API.
But anyone using the API will need to handle a message called `CreatePhraseRequest` which is indeed quite weird.